### ISSUE TITLE
Unlock Background layer 

### ISSUE THREAD

**Issue by konri** [Agamnentzar/ag-psd#47]
Hi team, 
I have an issue when generating PSD file with layers, that the last one (first in ag-psd children) is locked when the image is full-one color. But when I add small dot (even in 99% opacity) to that one-colored image, photoshop won't lock it. 

![147799307_1383061202029899_5367685294078409653_n](https://user-images.githubusercontent.com/3826473/107216617-50f0a780-6a0d-11eb-9be8-45bf0afc13a5.png)

example of Background:
![3BACKGROUND](https://user-images.githubusercontent.com/3826473/107216689-682f9500-6a0d-11eb-822c-b269ce00548a.png)

I tried add additional metadata for layer but still the same effect.  
```
{
          name,
          canvas: createCanvasFromImage(parsedImages[index]),
          opacity,
          protected: {
            transparency: false,
            composite: false,
            position: false
          },
          transparencyProtected: false,
          blendMode: blendMode.value
        }
```

When I open psd file in photoshop, unlock the layer and save, re-open the padlock is missing, so I believe that information is saved to psd file.  

I tried some workaround to duplicate the background layer, read once again buffer, delete children and save new configuration but also without a success:
```
      const children = layers.map((layer, index) => {
        const {name, opacity, blendMode} = layer;
        return {
          name,
          canvas: createCanvasFromImage(parsedImages[index]),
          opacity,
          protected: {
            transparency: false,
            composite: false,
            position: false
          },
          transparencyProtected: false,
          blendMode: blendMode.value
        }
      });

      const firstElement = {
        ...children[0],
        name: 'copyBG'
      }
      const psd = {
        width,
        height,
        children: [firstElement, ...children],
        canvas: fullCanvas
      };
      const outputFile = `${outputName}${folderName}_${sequence}.psd`;
      const generatePsdFile = generatePsd(psd);
      const readFromGeneratedPsd = readPsdFile(generatePsdFile);
      readFromGeneratedPsd.children.shift();
      console.log(`File: ${outputFile} generated`);
      const outputPath = `${output}/${outputFile}`;
      const buffer = writePsdBuffer(readFromGeneratedPsd);
      fs.writeFileSync(outputPath, buffer);
```
Thanks for any tips :)

**Comment by Agamnentzar** [comment]
Photoshop automatically treats bottom layer as background and locks it's alpha if the layer doesn't have any transparent pixels.

I added `noBackground` write option (use: `writePsdBuffer(readFromGeneratedPsd, { noBackground: true });`) to force it to never treat bottom layer as background layer and released it in version `11.5.0`.

Hope this solves your issue.

**Comment by konri** [comment]
Awesome! Thank you very much for your very quick help, it's working as expected.
